### PR TITLE
ISPN-13243 Log as debug level the system properties

### PR DESCRIPTION
**Pull request by diegolovison**
https://issues.redhat.com/browse/ISPN-13243

### CODE REVIEW

File: server/runtime/src/main/java/org/infinispan/server/Bootstrap.java [hunk 1]
```diff
@@ -200,6 +201,16 @@ private void logJVMInformation() {
          for(URL url : cl.getURLs()) {
             logger.fine("JAR: " + url);
          }
+         logger.fine(String.format("Starting server with system properties:%s", getSystemProperties()));
       }
    }
+
+   private String getSystemProperties() {
```
[the remark this answers is not in the document]
Comment (ryanemerson) [reply server/runtime/src/main/java/org/infinispan/server/Bootstrap.java]: That's still a log entry per property with each property pre-pended with "TIMESTAMP> DEBUG (main) [BOOT]"

[the remark this answers is not in the document]
Comment (diegolovison) [reply server/runtime/src/main/java/org/infinispan/server/Bootstrap.java]: Should I roll back and add the string concatenation method?

[the remark this answers is not in the document]
Comment (ryanemerson) [reply server/runtime/src/main/java/org/infinispan/server/Bootstrap.java]: That would be my preference

**Comment by tristantarrant** [comment]
Pushed a different implementation https://github.com/infinispan/infinispan/commit/194705715850f9e97959f5c40b7b7646344b2eaa

**Pull request closed without merging**